pipeline/resources.yml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: tekton.dev/v1alpha1
+kind: PipelineResource
+metadata:
+  name: repo
+spec:
+  type: git
+  params:
+    - name: url
+      value: https://github.com/raynalee4/training.git
```

Comment:
I think personal repo should be replaced with mlperf repo or fork of mlperf repo on AICoE org.

Reply:
I have created https://github.com/AICoE/mlperf-training

Reply:
If I do this, the Dockerfile is wrong. If you want me to merge my personal fork into AICoE I can do that.

Reply:
Kindly make changes here https://github.com/AICoE/mlperf-training/blob/master/image_classification/tensorflow/Dockerfile and a pull request to the repo

Reply:
https://github.com/AICoE/mlperf-training/pull/1